components/CoverletterInformation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <address>
+    <div 
+      class="company">
+      <span 
+        v-if="title"
+        v-text="title"/>
+    </div>
+    <div 
+      class="department">
+      <span 
+        v-if="department"
+        v-text="department"/>
+    </div>
+    <div 
+      class="name">
+      <span 
+        v-if="name.speech"
+        v-text="name.speech"/>
+      <span 
+        v-if="name.first"
+        v-text="name.first"/>
+      <span 
+        v-if="name.last" 
+        v-text="name.last"/>
+    </div>
+    <div 
+      class="streetHousenumber">
```

Comment:
these v-if below seems to be nonsense. Either put it up here (if there really is case where the street is not existing) or remove them. (add v-if="address")